src/client/screens/OrganizationAdminScreen.tsx

```diff
@@ -126,7 +132,13 @@ const OrganizationAdminScreen = ({ organization, user, organizationProjects }: S
           <Box>
             <Flex sx={style.header}>
               <Box>
-                <Heading as="h3">{organization.resource.name}</Heading>
+                <Heading
+                  as="h3"
+                  onClick={() => history.push(`/o/${organizationSlug}`)}
+                  sx={style.orgName}
+                >
+                  {organization.resource.name}
+                </Heading>
```

Comment:
We can use a `<Link>` here too, and it will eliminate the need for specialized styling as a bonus:
```suggestion
                <Heading
                  as="h3"
                >
                  <Link to={`/o/${organizationSlug}`}>{organization.resource.name}</Link>
                </Heading>
```